src/Microsoft.AspNet.FileProviders.Common/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyMetadata("Serviceable", "True")]
```

Comment:
This package doesn't really produce an assembly right? If no assembly, don't need this attribute.